code/modules/mob/living/carbon/human/human.dm

```diff
@@ -9,6 +9,7 @@
 	var/embedded_flag	  //To check if we've need to roll for damage on movement while an item is imbedded in us.
 	var/obj/item/weapon/rig/wearing_rig // This is very not good, but it's much much better than calling get_rig() every update_canmove() call.
 	mob_size = 9//Based on average weight of a human
+	var/malf_control = 0 // for IPC's only

```

Comment:
I wonder.

Would it be better to put this into the AI's antagonist datum? Or somewhere there? Something specific to the AI would be better, even better if it's the AI antagonist datum. Because cluttering up mob variables is bad, specially if they're very niche usage.